src/primitives/transaction.h

```diff
@@ -283,18 +295,14 @@ class CTransaction
     CTransaction(CMutableTransaction &&tx);
 
     CTransaction(const CTransaction& tx) = default;
```

Comment:
this was added in 454aa378e18ffa716a0c9ee69f9647588bcfaf72 to remove the `-Wdeprecated-copy` warning, due to the usage of the implicit CTransaction copy-ctor inside CMerkleTx constructor.
But since #2044 `CMerkleTx` no longer inherits from `CTransaction`, so we can remove it now.

Reply:
👍 . Will add it in #2201 that is the next in the line.